internal/resources/credential/resource_credential.go

```diff
@@ -38,12 +38,6 @@ var credentialSchema = map[string]*schema.Schema{
 	},
 	common.MetaKey: common.Meta,
 	specKey:        credSpec,
-	statusKey: {
```

Comment:
Can you paste the example plan in which case this field is showing updates? I understand this field is not really useful unless some automations rely on it to make some decision

Reply:
I am resolving this as we already discussed it